### PR TITLE
Cleanup of pathhandling and 2021 model

### PR DESCRIPTION
**Pull request by mihofer**

### CODE REVIEW

File: omc3/model/model_creators/lhc_model_creator.py [hunk 1]
```diff
@@ -85,13 +85,12 @@ def get_correction_check_script(
 
     @classmethod
     def prepare_run(cls, accel: Lhc) -> None:
-        print(f"year: {accel.year}")
-        if accel.year in ["2018", "2022"]:  # this should be handled by the fetcher
-            symlink_dst = Path(f"{accel.model_dir}/acc-models-lhc")
-            print(f"symlink dst: {symlink_dst}")
+        if accel.year in ["2018", "2022"]:  # these should be handled by the fetcher
```
Comment (awegsche) [inline omc3/model/model_creators/lhc_model_creator.py:88]: you _introduced_ a typo. not that it matters, just saying

File: omc3/optics_measurements/coupling.py [hunk 2]
```diff
@@ -109,12 +105,12 @@ def calculate_coupling(meas_input, input_files, phase_dict, tune_dict, header_di
     # old Cminus
     C_old = 4.0 * tune_sep * np.mean(np.abs(f1001))
     header_dict["OldCminus"] = C_old
-    LOG.info(f"abs OldCminus = {C_old}, tune_sep = {tune_sep}")
+    LOG.info(f"abs OldCminus = {C_old}, tune_sep = {tune_sep}, from Eq.1 in PRSTAB 17,051004")
```
Comment (fsoubelet) [inline omc3/optics_measurements/coupling.py:108]: 👍

**Review by fsoubelet — COMMENTED** [review]
Column names constants in coupling.py should be from the `constants` module. The `COLS_TO_KEEP` should also be defined from constants.

**Review by fsoubelet — APPROVED** [review]
I suggest we move on with this @mihofer @awegsche and take care of #320 before some final cleanup

**Comment by mihofer** [comment]
Fine by me, but I can't approve as PR opener.

**Review by awegsche — APPROVED** [review]

**Pull request merged**